src/PresentationalComponents/RulesTable/RulesTable.js

```diff
@@ -160,12 +160,15 @@ const RulesTable = ({ isTabActive }) => {
   }, []);
 
   useEffect(() => {
-    dispatch(
-      updateRecFilters({
-        ...filtersInitialState.recState,
-        ...getDefaultImpactingFilter(hasEdgeDevices),
-      })
-    );
+    if (!isLoading) {
```

Comment:
The `isLoading` should probably be a dependency of this effect, or?

Reply:
I tried running with isLoading added into dependencies. It seems if I add it, the bug still exists. But, if I remove it, the bug is gone.

Reply:
What is this `useEffect` trying to do? 

If the `isLoading` is used it should be a dependency it not working correctly if it is a dependency might point to a general issue with the flow in the component states.